### PR TITLE
Allow passing bn_code to Request#checkout! via opts

### PR DESCRIPTION
**Pull request by ronail**
Please refer to the original pull request [here](https://github.com/nov/paypal-express/pull/84)

### CODE REVIEW

**Comment by ianfleeton** [comment]
Is this a feature you are wanting to use? If so I'll add a test and then merge it in. Cheers.

File: lib/paypal/express/request.rb [hunk 1]
```diff
@@ -45,11 +45,17 @@ def transaction_details(transaction_id)
         Response.new response
       end
 
-      def checkout!(token, payer_id, payment_requests)
+      def checkout!(token, payer_id, payment_requests, opts = {})
```
Comment (korny) [inline lib/paypal/express/request.rb:48]: You could use keyword arguments here:

```ruby
def checkout!(token, payer_id, payment_requests, bn_code: nil)
```

**Comment by ianfleeton** [comment]
Thanks @korny good suggestion, we're assuming Ruby 2+ now.

**Comment by ianfleeton** [comment]
I've merged but then undone it again. During testing I noticed that PayPal has pulled the offer. Let me know if that's not the case. Cheers.

**Pull request merged**